network-api/networkapi/templates/pages/libraries/detail_page.html

```diff
@@ -0,0 +1,77 @@
+{% extends "pages/libraries/base.html" %}
+{% load i18n wagtailcore_tags wagtailimages_tags %}
+
+{% block library_content %}
+    <div class="tw-grid tw-grid-cols-1 tw-grid-rows-[auto_auto_1fr] medium:tw-grid-cols-[12rem_1fr_14rem] large:tw-grid-cols-[14rem_1fr_16rem] xlarge:tw-grid-cols-[14rem_1fr_16rem] tw-gap-12">
+
+        <div id="breadcrumbs-container" class="medium:tw-col-start-2 medium:tw-col-end-4">
+            {% block breadcrumbs %}{% endblock breadcrumbs %}
```

Comment:
We need a breadcrumbs block because each page gets their own breadcrumbs template tag. I might be able to unify this, but it will be done in a separate PR.